### ISSUE TITLE
Sorting the applications based on their labeling

### ISSUE THREAD

**Issue by mirunaioana-msc** [code4romania/onghub#477]
### **Current context**

Currently the applications made available for the NGO Admins are displayed being sorted alphabetically. 

### **Expected behavior** 

The applications should be sorted first by their label and then alphabetically, the order of the labels being the following: "Available", "Coming soon", and then the rest of labels, order which shall be discussed after issue #476 will be closed.
